README.md

```diff
@@ -156,3 +162,31 @@ It can be configured to instead allow login based on an external LDAP directory.
   - `virgo-tomcat-server-3.6.4.RELEASE/configuration/login.conf`
 
 When the server starts with the reconfigured portal, it will automatically create new accounts for valid LDAP logins.
+
+
+## Change Port Number (optional)
+* Change line#12 in the `site.properties` file present in the `/ApromoreEE/ApromoreCore/Apromore-Assembly/virgo-tomcat-server-3.6.4.RELEASE/repository/usr` directory.
```

Comment:
Try Avoiding mentioning line numbers in the doc, as they keep changing.
Be specific on the parameter name.